api/src/org/apache/cloudstack/api/command/admin/config/UpdateCfgCmd.java

```diff
@@ -145,6 +156,9 @@ public void execute() {
             if (getAccountId() != null) {
                 response.setScope("account");
             }
+            if (getImageStoreId() != null) {
+                response.setScope("imagestore");
+            }
```

Comment:
Please remove this check and adding ``validations = PositiveNumber`` to the ``imageStoreId @Parameter`` annotation.